app/entity_helper.py

```diff
@@ -0,0 +1,84 @@
+from dataclasses import is_dataclass
+from datetime import datetime
+from typing import Any
+from functools import wraps
+
+
+class EntityHelper:
+    __placeholder_symbol = "?"
+
+    def __init__(self, entity_type: type, table_name: str) -> None:
+        if not is_dataclass(entity_type):
+            raise ValueError(f"EntityHelper is able to work with "
+                             f"dataclasses only. {type} is not a dataclass")
+        self.__entity_type: type = entity_type
+        self.__table_name: str = table_name
+        self.__entity_type_fields: tuple = tuple(
```

Comment:
no need to annotate types for variables if you understand what will be in them from their initialization

Reply:
I am aware of that, but I don't want to depend on the `__init__`'s parameters type hints when I analyze my code. I've seen no rules that prevent me from doing this kind of hinting, so I tend to do it, at least for my fields inside of my classes. It is easier to read the code this way, so until there are no company rules against this style, I want to follow it